.github/workflows/maven.yaml

```diff
@@ -82,3 +89,55 @@ jobs:
         with:
           name: container-logs
           path: "**/container-logs/**/*.log"
+
+  older_kafka_matrix:
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        version: [3.8.0]
```

Comment:
We'll need to have a note somewhere to rev this.  How many previous version do we actually want to test with?

Reply:
I'm not sure we really need multiple older versions. Possibly just 3.9.0 (as the LTS IIRC with ZK support)?

Reply:
I can see an argument for testing `4.0` explicitly in the matrix as well in that it gives us confidence that things work across versions regardless of where main is.